### PR TITLE
py-docutils: added select group (including new docutils_select port)

### PR DESCRIPTION
**Pull request by catap**
#### Description

Let make possibly to enduser to decide which version of `rst2man` should be used by default.

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on
macOS 11.0.1 20B50
Xcode 12.2 12B45b

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-docutils/Portfile [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 
-PortSystem 1.0
-PortGroup python 1.0
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           select 1.0
 
 name                py-docutils
 version             0.16
+revision            2
```
Comment (mf2k) [inline python/py-docutils/Portfile:9]: The `revision` should only be 1 here.

**Review by mf2k — CHANGES_REQUESTED** [review]

File: python/py-docutils/Portfile (hunk 1 above)
Comment (catap) [reply python/py-docutils/Portfile]: @mf2k thanks, fixed!

File: python/py-docutils/Portfile [hunk 2]
```diff
@@ -26,14 +28,36 @@ checksums           md5    44952782107930ddfcd37ae48eee0857 \
 python.versions     27 35 36 37 38 39
 
 if {$subport ne $name} {
-    depends_run         port:py${python.version}-roman
-    
+    depends_run         port:py${python.version}-roman \
+                        port:docutils_select
```
Comment (mf2k) [inline python/py-docutils/Portfile:32]: Nitpick. Please sort these alphabetically.

**Review by mf2k — CHANGES_REQUESTED** [review]

File: python/py-docutils/Portfile (hunk 2 above)
Comment (catap) [reply python/py-docutils/Portfile]: @mf2k fixed

**Review by mf2k — APPROVED** [review]

**Comment by catap** [comment]
@mf2k that should we do now? :)

**Comment by catap** [comment]
@mf2k can you merge it? :)

**Comment by catap** [comment]
Just rebased to last master

**Comment by catap** [comment]
@ryandesign can I ask your attention here? :) Maybe you can forward it to someone?

**Comment by catap** [comment]
rebased to last master

**Comment by catap** [comment]
@kencu and I have one more very old PR that is very useful when you try to develop for example Varnish on machine :) 

May I ask you to check this one? ;)

**Comment by kencu** [comment]
This looks reasonable.

This port appears to be actively maintained by one of MacPorts top dudes, @jmroot, and he's here many times each day.

I'm not certain what the holdup, if any, might be but can understand my reluctance to get my finger bit...

**Pull request merged**